lib/src/base-styles/_nx-forms.scss

```diff
@@ -6,10 +6,6 @@
  */
 @import 'nx-text-input';
```

Comment:
Just noticed this import is causing the nx-text-input styles to be present twice.  It should be removed

Reply:
> Just noticed this import is causing the nx-text-input styles to be present twice. It should be removed

https://github.com/sonatype/sonatype-react-shared-components/pull/57/commits/0eda4995d66fc3c02b64aea0a476fbdf5dffdc79